core/crypto/client_tca.go

```diff
@@ -190,17 +189,17 @@ func (client *clientImpl) getTCertFromExternalDER(der []byte) (tCert, error) {
 		// Compute temporary public key
 		tempX, tempY := client.enrollPrivKey.PublicKey.ScalarBaseMult(k.Bytes())
 		tempSK.PublicKey.X, tempSK.PublicKey.Y =
-		tempSK.PublicKey.Add(
-			client.enrollPrivKey.PublicKey.X, client.enrollPrivKey.PublicKey.Y,
-			tempX, tempY,
-		)
+			tempSK.PublicKey.Add(
```

Comment:
@srderson I have checked this file and that is the format generated by goimports.  I tried removing the indent a run goimports again an the indent is generated automatically by goimports.